samples/bluetooth/fast_pair/input_device/boards/nrf54h20dk_nrf54h20_cpuapp.overlay

```diff
@@ -20,13 +20,22 @@
 			reg = <0x100000 DT_SIZE_K(908)>;
 		};
 
-		storage_partition: partition@1e3000 {
-			reg = <0x1e3000 DT_SIZE_K(20)>;
-		};
+		/* Leaving here an empty gap from 0x1e3000 to 0x1e3fb0.
+		 * We want to have a bt_fast_pair_partition preceding the
+		 * storage_partition. We also need a storage partition to
```

Comment:
I would keep comments related to the settings partition together with the settings partition definition. I am not sure if we should mention that `we want to have bt_fast_pair_partition preceding the storage_partition` (it's just matter of preference - not a requirement, it's also clearly visible here)